src/calculateRentalCost.js

```diff
@@ -3,8 +3,30 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_CAR_C0ST = 40;
+
+  const LONG_TERM = 7;
+  const LONG_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_DISCOUNT = 20;
+
+  let allResult = 0;
```

Comment:
The variable 'allResult' is not descriptive of its purpose. A more descriptive name such as 'totalCost' would better represent its role in the function.